src/js/config.js

```diff
@@ -0,0 +1,127 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://raw.githubusercontent.com/GPII/first-discovery-server/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion");
+var ajv = require("ajv");
+var $ = fluid.registerNamespace("jQuery");
+var gpii = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.firstDiscovery.server.config.oauth2.clientCredential");
+
+/**
+ * Attemps to load a config file, will return an empty object {} if the file
+ * cannot be located.
+ *
+ * @param file {String} - path to the config file
+ * @return {Object} - the JSON object representation of the JSON file, default is {}
+ */
+gpii.firstDiscovery.server.config.getConfigFile = function (file) {
+    var config = {};
+    try {
+        config = require(file);
+    } catch (e) {
+        console.log(file + " not found.");
+    }
+    return config;
+};
+
+/**
+ * Transforms an ENVMap into a config object
+ *
+ * @param ENVMap {Object} - a map of Environment variables that is transformed into a
+ *                          configuration object.
+ * @returns {Object} - the configuration object
+ */
+gpii.firstDiscovery.server.config.getENV = function (ENVMap) {
```

Comment:
I think capitalisation of "ENV" can be "Env" here and throughout this implementation - it isn't any kind of acronym, after all